app-crypt/veracrypt/veracrypt-1.19-r1.ebuild

```diff
@@ -45,6 +45,15 @@ PATCHES=(
 	"${FILESDIR}/${P}-remove-packaging-from-makefile.patch"
 )
 
+src_prepare() {
+	default
+	WX_BASENAME=$( ${EROOT}usr/bin/wx-config --basename )
```

Comment:
`${EROOT}` is not available in src_* phases. Just rely on PATH working correctly, and call the executable by name.